src/core/syscalls/business_logic_syscall_handler.rs

```diff
@@ -469,30 +460,30 @@ mod tests {
         let mut vm = vm!();
         add_segments!(vm, 3);
 
-        // insert syscall_ptr
-        let syscall_ptr = relocatable!(2, 0);
+        // insertion order
 
-        let version = bigint!(1);
-        let account_contract_address = bigint!(1);
-        let max_fee = bigint!(2);
-        let signature_len = bigint!(1);
-        let signature = Relocatable::from((3, 0));
-        let transaction_hash = bigint!(1);
-        let chain_id = bigint!(1);
-        let nonce = bigint!(1);
+        //  syscall_ptr
+        //  version
+        //  account_contract_address
+        //  max_fee
+        //  signature_len
+        //  signature
+        //  transaction_hash
+        //  chain_id
+        //  nonce
 
         memory_insert!(
             vm,
             [
-                ((1, 0), syscall_ptr),
-                ((2, 0), version),
-                ((2, 1), account_contract_address),
-                ((2, 2), max_fee),
-                ((2, 3), signature_len),
-                ((2, 4), signature),
-                ((2, 5), transaction_hash),
-                ((2, 6), chain_id),
-                ((2, 7), nonce)
+                ((1, 0), (2, 0)),
```

Comment:
Please add comments to specify what field is each number

Reply:
move the comments next to the values to mark which one is which